uaclient/api/exceptions.py

```diff
@@ -0,0 +1,23 @@
+from uaclient.exceptions import (
+    AlreadyAttachedError,
+    BetaServiceError,
+    ContractAPIError,
+    EntitlementNotEnabledError,
+    EntitlementNotFoundError,
+    FullAutoAttachFailureError,
+    NonAutoAttachImageError,
+    UrlError,
+    UserFacingError,
+)
+
```

Comment:
@aciba90 We are importing exceptions here to use for the api calls. So instead of importing from `uaclient.exceptions`, you could use `uaclient.api.exceptions`.